editor/src/components/inspector/common/css-utils.ts

```diff
@@ -2055,7 +2055,7 @@ export function cssColorToChromaColor(c: CSSColor): Either<string, Chroma.Color>
   } else if (isHex(c)) {
     return right(Chroma.hex(c.hex))
   } else if (isHSL(c)) {
-    return right((Chroma.hsl as any)(c.h, c.s, c.l, c.a))
+    return right((Chroma.hsl as any)(c.h, c.s / 100.0, c.l / 100.0, c.a))
```

Comment:
is this `c.s / 100.0` different from `c.s / 100` in javascript?

Reply:
I don't believe so, they're apparently equal, it just seemed like the right thing to do given that it's going to be double-like.